pom.xml

```diff
@@ -164,11 +161,9 @@
     <maven-surefire-plugin.argLine>-Xmx8192m -XX:+HeapDumpOnOutOfMemoryError</maven-surefire-plugin.argLine>
     <maven-surefire-plugin.argLineAccessArgs />
     <maven-surefire-plugin.version>3.5.2</maven-surefire-plugin.version>
-    <maven-surefire-report-plugin.version>${maven-surefire-plugin.version}</maven-surefire-report-plugin.version>
     <maven-war-plugin.version>3.4.0</maven-war-plugin.version>
     <maven.compiler.createMissingPackageInfoClass>false</maven.compiler.createMissingPackageInfoClass>
     <maven.core.version>3.9.9</maven.core.version>
-    <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
```

Comment:
This should not be removed.

https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#redirectTestOutputToFile